rpcs3/Emu/Cell/Modules/cellSysutil.cpp

```diff
@@ -283,18 +283,36 @@ s32 cellSysutilUnregisterCallback(u32 slot)
 	return CELL_OK;
 }
 
-s32 cellSysCacheClear(void)
+s32 cellSysCacheClear()
 {
-	cellSysutil.todo("cellSysCacheClear()");
+	// Checks if the param exists, else we quit
```

Comment:
Log command shouldn't be removed.

Reply:
Okay. I'll push a new version tonight after I get back from work.